### ISSUE TITLE
disconnecting

### ISSUE THREAD

**Issue by middo11** [Paliverse/DualSenseX#16]
the notification of connected/reconnected/ing keeps flashing and sometimes it does the sound of connected but it doesnt seem to apply any settings. it does say connected in the box. have reinstalled it twice and tried 2 cables. could be a dodgy controller but works find without the app

![Screenshot (39)](https://user-images.githubusercontent.com/89025176/129575279-c4652868-5f63-40ba-9598-030a32990e4a.png)
![Screenshot (40)](https://user-images.githubusercontent.com/89025176/129575280-6e7439f7-38b6-4b2b-ae1c-86cfd89433f4.png)
![Screenshot (41)](https://user-images.githubusercontent.com/89025176/129575283-6da451b9-c197-4b7c-9ede-e897f2a51909.png)

**Comment by Paliverse** [comment]
hmm that's weird, can you try to delete the save file located in: C:\Temp\DualSenseX\DualSenseX_SaveFile.xml

Start DualSenseX after deleting.
Let me know what happens.

**Comment by middo11** [comment]
ive tried it before and it worked for a few. it seems to work but does it mean ive got to delete that file every time?

**Comment by Paliverse** [comment]
not really, but i'm trying to see how i can figure out what the source of the problem is, and it sucks theres not much to go on in trying to reproduce the issue on my end. ill keep the issue open for now

**Comment by middo11** [comment]
ok thats cool. just seems like its the save file as when i shut computer down and load it up again nothing happens.

**Comment by Paliverse** [comment]
Are you running the latest version 1.3.3?

**Comment by middo11** [comment]
yeah sir, im starting to think it my be firewall/antivirus ie bitdefender
